packages/example/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import { Layout } from 'antd';
+import './App.css';
+import { Outlet } from 'react-router-dom';
+import { NavigationMenu } from './components/NavigationMenu';
+
+const { Header, Content, Footer } = Layout;
+
+export function App(): JSX.Element {
+  return (
+    <Layout className="layout">
+      <Header className="header" style={{ height: 67 }}>
```

Comment:
I'm not aware of the framework choice but you decided to keep classes so please don't use style property